src/components/LeverageSlider/LeverageSlider.tsx

```diff
@@ -81,6 +81,11 @@ export function LeverageSlider(p: Props) {
 
 function LeverageSliderHandle({ value, dragging, index, keyValueMap, ...restProps }: HandleProps) {
   const displayValue = keyValueMap[value || 0] ?? DEFAULT_LEVERAGE_KEY;
+
+  useEffect(() => {
+    document.body.classList.toggle("dragging", dragging);
```

Comment:
I worry about what might happen if the component were initially rendered with the dragging={false} prop. There could be desynchronization between the prop value and the class. What if we better add and remove the class name explicitly?